src/Restaurant.js

```diff
@@ -1,17 +1,40 @@
-import React from "react";
+import React, { useState } from "react";
 
-const Restaurant = () => {
-  const pizzas = 0;
+export const Restaurant = () => {
   return (
-    <div>
-      <h3>Restaurant Orders</h3>
-      <ul>
-        <li>
-          Pizzas: {pizzas} <button className="btn btn-primary">Add</button>
-        </li>
+    <div className="restuarant-container">
+      <h3 className="restaurant-header">Restaurant Orders</h3>
+      <ul className="list-of-food">
+        <Order food={"Pizza"} />
+        <Order food={"Salad"} />
+        <Order food={"Chocolate cake"} />
       </ul>
     </div>
   );
 };
 
+function RestaurantButton(props) {
+  return (
+    <button onClick={props.setOrdersFunction} className="btn btn-primary">
+      Add
+    </button>
+  );
+}
+
+function Order(props) {
+  const [orders, setOrders] = useState(0);
```

Comment:
if the variable is a number type rather than an array, it is better to name it `order`, `orders` implies it is an array.